fuel/app/classes/helper/common.php

```diff
@@ -0,0 +1,40 @@
+<?php
+class Helper_Common
+{
+		public static function format_time($timestamp)
+		{
+			return Date::forge($timestamp)->format('%m/%d/%Y %H:%M');
+		}
+
+		public static function get_hot_items($array,$key,$limit = 3)
+		{
+			if( ! is_array($array))
+			{
+				return array();
+			}
+
+			usort($array,Helper_Common::call_compare_func($key));
+			return array_slice($array,0,$limit);
+		}
+
+		public static function call_compare_func($key)
```

Comment:
@kondo-takuto-mulodo : 
get_hot_items function will take the first items follow $limit parameter of item list what is arranged follow call_compare_func function.
It only is sorting and collecting of array, it does not depend on model or database. So I want to use It for other places althought we only got Message page at the present. 

If I don't set it as public function, we can't call it in controller. So have you some advice about it ?